contentcuration/contentcuration/frontend/shared/mixins/draggable/handle.js

```diff
@@ -23,35 +21,21 @@ export default {
       type: Boolean,
       default: false,
     },
+    draggableType: {
+      type: String,
+      default: DraggableTypes.HANDLE,
+    },
     effectAllowed: {
```

Comment:
Following up on my earlier comment, not sure how much more complex it would be to allow Array types here?